pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1630,6 +1628,11 @@ Question:
       type: array
       items:
         $ref: '#/ValidationFailure'
+    columnSpan:
+      type: integer
+      minimum: 1
+      default: 1
+      description: The number of columns this question should occupy in a tabular display
```

Comment:
Not clear here because of how github displays diffs, but this is being applied to question blocks.
This becomes useful for composites questions. columnSpan of 1 means child questions are stacked on top of each other.  If columnSpan is not used, composite will use one column per child.